uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
I have no idea why you did that instead of putting these removals inside the regform under an AT_THE_CON check. :P